samples/FunctionApp/HttpTriggerWithFunctionContextLogger/HttpTriggerWithFunctionContextLogger.cs

```diff
@@ -8,9 +8,9 @@
 
 namespace FunctionApp
 {
-    public static class Function4
+    public static class HttpTriggerWithFunctionContextLogger
     {
-        [Function("Function4")]
+        [Function(nameof(HttpTriggerWithFunctionContextLogger))]
```

Comment:
I am more inclined to leave this as `HttpTriggerStatic` or `HttpTriggerSimple` ?